backend/app/Http/Controllers/User/ShowIndividualUserController.php

```diff
@@ -28,7 +38,118 @@ public function __construct(Gate $gate)
      */
     public function __invoke(int $user_id): View|Factory
     {
-        $userInfo = UserInfo::where('user_id', $user_id)->first();
-        return view('user.individual', ['gate' => $this->gate, 'userInfo' => $userInfo]);
+        /**
+         * @var User|UserInfo|UUMajor
+         */
+        $user = DB::table('user_infos')
+            ->where('user_id', $user_id)
+            ->join('users', 'user_infos.user_id', '=', 'users.id')
+            ->leftJoin('u_u_majors', 'user_infos.u_u_major_id', '=', 'u_u_majors.id')
+            ->select('users.id', 'users.name', 'users.profile_photo_path', 'users.user_role_id', 'user_infos.status', 'user_infos.grade', 'user_infos.gender', 'user_infos.university_meta', 'user_infos.company_meta', 'user_infos.birth_area_id', 'user_infos.live_area_id', 'user_infos.group_affiliation', 'user_infos.is_publish_birth_day', 'user_infos.birth_day', 'user_infos.hobbies', 'user_infos.interests', 'user_infos.motto', 'user_infos.slack_name', 'user_infos.discord_name', 'user_infos.line_name', 'user_infos.github_id', 'u_u_majors.name as uu_major', 'u_u_majors.faculty_id as uu_faculty')
+            ->first();
+
+        /**
+         * @var Area
+         */
+        $userAreas = Area::whereIn('id', [$user->birth_area_id, $user->live_area_id])->get();
+        // 在住と出身が同じだった場合、返り値1つなので
+        $userBirthArea = $userAreas->first(fn (Area $area) => $area->id === $user->birth_area_id);
+        $userLiveArea = $userAreas->first(fn (Area $area) => $area->id === $user->live_area_id);
+        // userにbirthとliveメソッドを追加
+        $user->birth_area = $userBirthArea->prefecture_code->label() . $userBirthArea->municipality;
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property App\Models\User|App\Models\UserInfo|App\Models\UUMajor::$birth_area.